Expensify/App
# Set up Avatars for Workspace Chats

### Details

This PR includes the avatars for workspace chats (for both admins and workspace members) and shows the workspace chats when the user has the `policyExpenseChat` beta. I also included fixes for some minor issues I found (like duplicate rows on the Search Page and the user name not included in the second line of the chat header).

Additionally, I created a Web-E PR to fix a few issues I found while I was implementing the workspace chats: https://github.com/Expensify/Web-Expensify/pull/33166

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/195471

### Tests / QA

Pre-requisites:
  - Create an account which will be the admin
  - Create 2 or more accounts which will be the workspace members 
  - Add the created accounts to `policyExpenseChat` beta

Steps:
1. With the admin account create a Workspace and invite the workspace members.
2. Check the workspace chats were created for each workspace member in the Search Page and send them a message.
3. Login with the workspace members and verify there's only one workspace chat and check the message of the admin was received.

- [x] (Test) Verify that no errors appear in the JS console
- [ ] (QA) Verify that no errors appear in the JS console

### Tested On


### Screenshots


#### Web

User:
<img width="900" alt="Screen Shot 2022-02-23 at 18 41 44" src="https://user-images.githubusercontent.com/6829422/155435309-df1a152e-fa68-43a1-aba0-7f61d5ee5672.png">

Admin:
<img width="900" alt="Screen Shot 2022-02-23 at 18 43 15" src="https://user-images.githubusercontent.com/6829422/155435306-35de979e-b4c3-44aa-9ebc-2573c694041a.png">


#### Mobile Web

User (with search value):
<img width="400" alt="Screen Shot 2022-02-23 at 18 49 21" src="https://user-images.githubusercontent.com/6829422/155435848-7c5fbcbd-853a-4b5d-8f5f-6c740e5fb848.png">

Admin (with search value):
<img width="400" alt="Screen Shot 2022-02-23 at 18 30 34" src="https://user-images.githubusercontent.com/6829422/155435384-56883e74-435b-452b-b6da-870fcb3ed77b.png">

#### Desktop

<img width="900" alt="Screen Shot 2022-02-23 at 18 52 38" src="https://user-images.githubusercontent.com/6829422/155436158-bc28a27b-f53c-449f-84ab-4084bb13a412.png">


#### iOS

<img width="400" alt="Screen Shot 2022-02-23 at 19 01 07" src="https://user-images.githubusercontent.com/6829422/155437156-56867e93-dca0-47e2-bc9f-f6451697d673.png">

<img width="400" alt="Screen Shot 2022-02-23 at 19 02 30" src="https://user-images.githubusercontent.com/6829422/155437157-84fc1883-68c9-4a5a-b5d1-61879f66e56b.png">


#### Android

<img width="400" alt="Screen Shot 2022-02-23 at 19 52 32" src="https://user-images.githubusercontent.com/6829422/155442233-745f8495-f520-4cc4-9484-0e777908298e.png">

<img width="400" alt="Screen Shot 2022-02-23 at 19 52 41" src="https://user-images.githubusercontent.com/6829422/155442237-1ee1252b-0947-4bb7-9fe2-6b64a5008c40.png">
